### ISSUE TITLE
validate output is hard to parse 

### ISSUE THREAD

**Issue by jbpoline** [psych-214-fall-2016/diagnostics-01#21]
At the moment, the validation output looks like :
9b8cf2548b507334aad5ffc0de9d323e2189a847 9b8cf2548b507334aad5ffc0de9d323e2189a847
9363868f0271dde318e5aca1c03cb76c870ac227 9363868f0271dde318e5aca1c03cb76c870ac227
...

could we improve the output ?

**Comment by jdtheiss** [comment]
The improved validate_data.py script now prints "[filename] has correct hash" or raises an error otherwise.
